### ISSUE TITLE
Merge/update not working properly

### ISSUE THREAD

**Issue by HenriqueMM2** [whiteblXK/ObjectBuilder#13]
I tried to update my client from 8.60 to 10.98 but I had several problems ingame, I will post pictures of the bugs. By the way with the regular spr and dat 10.98 client these problems do not occur.

![alguem](https://user-images.githubusercontent.com/52054292/159990656-5c13b8ec-4ccd-4bb6-9be0-6c68be756d2c.png)
![problem](https://user-images.githubusercontent.com/52054292/159990653-c14f58d0-d447-4ee3-ba90-f5fc7cfec364.png)

I recorded a video also demonstrating some problems caused by the merge.
https://youtu.be/FTgTKliBKBE

If you know of any way I can update my client without bugging like this, please let me know, or maybe there is something I can do to make it work normally.

In the last case I would have to add sprite by sprite in the 10.98 client

**Comment by ghost** [comment]
yes you better adding sprite by sprite because converting a version to another is not 100% working to just "convert and use", the code is not complete, there are LOTS of item flags/attributes that need to be recreated for the client to read them properly
